etc/tools/runsplitter/src/Runsplitter.cc

```diff
@@ -0,0 +1,151 @@
+/*
+ * Author: Paul Schuetze
+ *
+ * Purpose: Splitting .raw files from eudaq into several files
+ *
+ *
+ *
+ */
+
```

Comment:
In most files we are not doing this kind of comment header, style guide question? Reminds me: The Authors file should be updated anyhow.